pyuvsim/mpi.py

```diff
@@ -67,6 +67,7 @@ def start_mpi(block_nonroot_stdout=True):
         # For non-root ranks, do not print to stdout.
         # (Uncovered until we have multi-rank tests)
```

Comment:
Could you remove this parenthetical from the comment? These lines are probably covered now...